code-challenges.test.js

```diff
@@ -38,30 +35,84 @@ const object3 = { number: -7 }
 
 // b) Create the function that makes the test pass.
 
+const divisibleBy = (object) => {
+    if (object % 3 === 0 ) {
+        return `${object.number} is divisible by three.`
+    }else {
+        return `${object.number} is not divisible by 3`
+    }
+}
+console.log (divisibleBy(object1));
+console.log (divisibleBy(object2));
+console.log (divisibleBy(object3));
+
 // --------------------2) Create a function that takes in an array of words and returns an array with all the words capitalized.
 
 // Pseudo code:
-
+//input: string
+//output: modified string
+//process: using a fucntion we can tell this program to take the string as an inout and return it with a capitalized letter. using .map to iterate through array. Then using the charAt we can tell the program to look at a specified value, then apply .toUpperCase to capitalize, and using .slice and telling it where to slice and stop the uppercasing process.
```

Comment:
Almost! The `.slice` is not used to stop the uppercasing process, but it is used to create a substring. Giving `.slice` and argument of 1 like you did will return a string of everything in the original string except index 0. So in the map, you're returning each string with the character at index 0 capitalized, concatenated with the rest of the original string.